components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -101,11 +104,20 @@
                     buff.append(grantType + " ");
                 }
             }
-
             grants = buff.toString();
 
+            List<String> registeredScopeValidators = new ArrayList<String>();
+            String[] allowedValidators = client.getAllowedScopeValidators();
```

Comment:
Is there any possibility allowedValidators will be null? If there is a possibility of null, for iteration, could throw NPE, hence need to check before starting iteration

Reply:
In OAuthAdminService side, I have made sure that it will not return any null. 
anyway, I will add a check here also.